### ISSUE TITLE
[Betterbags Theme] Broken due to the theme stuff being officially released.

### ISSUE THREAD

**Issue by JourneyOver** [Witnesscm/NDui_Plus#33]
As noted in #29 about themes coming to betterbags.

Theming for betterbags has officially been released, which sadly has broken the betterbags skin from NDui+

**Comment by Witnesscm** [comment]
Thanks for your feedback, fixed in [11.0.0-alpha-3](https://github.com/Witnesscm/NDui_Plus/releases/tag/11.0.0-alpha-3).

However, the theme API provided by BetterBags can't be applied on elements such as icons, scrollbars, etc. So some elements still use the original method.

**Comment by JourneyOver** [comment]
> However, the theme API provided by BetterBags can't be applied on elements such as icons, scrollbars, etc

From my understanding more and more theme APIs will be provided as time goes on, right now Ciden is working on item buttons.

**Comment by JourneyOver** [comment]
@Witnesscm as of https://github.com/Cidan/BetterBags/releases/tag/v0.1.67 which just released about ~30-40 minutes ago it looks like icons can now be themed.

**Comment by JourneyOver** [comment]
Actually @Witnesscm it looks like with the release it completely broke betterbags again if you have the skin enabled in NDui+... the following error occurs:
```
...erface/AddOns/NDui_Plus/Modules/Skins/BetterBags.lua:53: attempt to index field 'ItemSlotBackground' (a nil value)
Count: 2

Call Stack:
[string "=(tail call)"]: ?
[string "@Interface/AddOns/NDui_Plus/Modules/Skins/BetterBags.lua"]:53: in function <...erface/AddOns/NDui_Plus/Modules/Skins/BetterBags.lua:48>
[string "@Interface/AddOns/NDui_Plus/Modules/Skins/BetterBags.lua"]:141: in function `Create'
[string "@Interface/AddOns/BetterBags/frames/item.lua"]:571: in function <Interface/AddOns/BetterBags/frames/item.lua:564>
[string "=[C]"]: ?
[string "@Interface/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ace/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:523: in function <...ace/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:516>
[string "=(tail call)"]: ?
[string "@Interface/AddOns/BetterBags/core/init.lua"]:215: in function <Interface/AddOns/BetterBags/core/init.lua:210>
[string "=[C]"]: ?
[string "@Interface/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:66: in function <...ace/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua:61>
[string "@Interface/AddOns/Masque/Libs/AceAddon-3.0/AceAddon-3.0.lua"]:523: in function `EnableAddon'
...
[string "@Interface/AddOns/Blizzard_UIParent/Mainline/UIParent.lua"]:402: in function `UIParentLoadAddOn'
[string "@Interface/AddOns/Blizzard_UIParent/Mainline/UIParent.lua"]:607: in function `MajorFactions_LoadUI'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_DragonflightLandingPage.lua"]:179: in function `SetUpMajorFactionList'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_DragonflightLandingPage.lua"]:172: in function `RefreshMajorFactionList'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_DragonflightLandingPage.lua"]:167: in function `RefreshOverlay'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_DragonflightLandingPage.lua"]:158: in function <...sionLandingPage/Blizzard_DragonflightLandingPage.lua:154>
[string "=[C]"]: in function `CreateFrame'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_DragonflightLandingPage.lua"]:122: in function `CreateOverlay'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_ExpansionLandingPage.lua"]:63: in function `RefreshExpansionOverlay'
[string "@Interface/AddOns/Blizzard_ExpansionLandingPage/Blizzard_ExpansionLandingPage.lua"]:35: in function <...pansionLandingPage/Blizzard_ExpansionLandingPage.lua:33>
```

The only way to fix it seems to be to go into the NDui+ options and completely disable the betterbags skin, as it's sadly not as simple as just going and deselecting it from the theme menu of betterbags.

**Comment by Witnesscm** [comment]
@JourneyOver fixed in [11.0.0-alpha-4](https://github.com/Witnesscm/NDui_Plus/releases/tag/11.0.0-alpha-4)
